driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTest.java

```diff
@@ -102,6 +102,10 @@ public abstract class UnifiedTest {
     private static final Set<String> PRESTART_POOL_ASYNC_WORK_MANAGER_FILE_DESCRIPTIONS = Collections.singleton(
             "wait queue timeout errors include details about checked out connections");
 
+    public enum Language {
+        JAVA, KOTLIN, SCALA
```

Comment:
Looks like `SCALA` is not used because we don't have a Scala unified test runner, and don't run unified tests via our Scala API. Let's remove this enum constant?